src/lib/deployments/naming/index.js

```diff
@@ -11,15 +11,17 @@ import Haikunator from "haikunator";
  * @return {String} The release name.
  */
 export function validateReleaseName(releaseName) {
+  const maxCharLen = 63;
   // Test string pattern
   const pattern = /^(?![0-9]+$)(?!.*-$)(?!-)[a-zA-Z0-9-]{1,63}$/g;
-  if (!pattern.test(releaseName)) throw new InvalideReleaseName();
+  if (!pattern.test(releaseName)) throw new InvalidReleaseName();
 
   // Test total char length
   const prefix = config.get("helm.releaseNamespace");
```

Comment:
wdyt if we fallback to default value here?
```suggestion
  const prefix = config.get("helm.releaseNamespace") || "";
```

Reply:
Do you think this will cause any downstream issues?  Should the namespace be the default k8s namespace, either `"default"` or "`astronomer`" instead of `""`?

Reply:
default/astronomer is not a good idea... may be @schnie has ideas in mind.

Reply:
I talked to @schnie and we thought it was best to add a default value into the `test.yaml`.  The releaseNamespace is set when the platform is launched via helm.  I think defaulting that prefix to anything in the code may cause more downstream issues later on.